pages/linux/wall.md

```diff
@@ -1,7 +1,6 @@
 # wall
 
 > Write a message on the terminals of users currently logged in.
-> Only available to super-user.
```

Comment:
@sbrl #1421 contains a previous discussion that `wall` does _not_ require super-user privileges.  Users can send and receive `wall` messages by default.  **If** a user disables the messages a super-user can override that and send them a message anyway.  The line is incorrect as stands and the conclusion of the ticket was that we should drop it overall.